Codegen/src/Generator.cs

```diff
@@ -269,7 +269,6 @@ public virtual void Generate(CompilationUnit compilationUnit, ColumnsLayout colu
             //Check if there is any error in diags
             if (compilationUnit.AllDiagnostics().Any(d => d.Info.Severity == Compiler.Diagnostics.Severity.Error))
             {
-                AnalyticsWrapper.Telemetry.TrackEvent(EventType.Generation, "Diagnostics Detected", LogType.Genration);
```

Comment:
What do you mean by 'so it means that if an error happens the Cobol develop cannot do anything.' ?
I've checked this part and in fact this is more an assert than a test because the `Generate` method should never be called on a failed compilation. Anyway in CLI, the `GenerationException` thrown here is caught, stored and reported through the `LoggingSystem`. See https://github.com/TypeCobolTeam/TypeCobol/blob/00f826451e652e03bce215ad092fc58ed5062143/CLI/src/CLI.cs#L562.